client/src/components/layouts/global.css

```diff
@@ -436,7 +436,7 @@ pre {
 
 .panel-primary > .panel-heading {
   background-color: transparent;
-  border-color: var(--color-primary);
+  border-color: var(--primary-color);
 }
```

Comment:
We could just add ` color: var(--primary-color);` here and we would not need the .panel-heading style in solution-viewer.css

Reply:
@ahmadabdolsaheb Seeing as `solution-viewer.css` is such a small file, should I just delete it, and move the CSS to `global.css`?

Reply:
yes please :)